code/src/components/Result.js

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+
+export const Result = (props) => {
+  return (
+    <div className="form-result">
```

Comment:
Use the same naming convention
```suggestion
    <div className="formResult">
```